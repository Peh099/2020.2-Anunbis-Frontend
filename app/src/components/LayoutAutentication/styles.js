import styled from 'styled-components';
import bgFGA from '../../assets/images/BG_FGA.png';

export const Background = styled.div`
  height: 100vh;
  background-image: url(${bgFGA});
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
`;

export const Base = styled.div`
  background-color: #ffd54f;
  height: min(500px, 70vh);
  width: min(800px, 70vw);
  border-radius: 20px;
  display: flex;
  justify-content: space-around;
`;

export const Imagem = styled.img`
  @media (max-width: 640px) {
    display: none;
  }
  height: min(500px, 70vh);
  width: min(50%, 400px);
  border-top-right-radius: 20px;
  border-bottom-right-radius: 20px;
  object-fit: cover;
`;

export const Main = styled.div`
  max-height: 500px;
  max-width: 400px;
`;
