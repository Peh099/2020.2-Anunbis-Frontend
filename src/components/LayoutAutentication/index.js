import React from "react";
import styled from 'styled-components';

const Background = styled.div`
    height: 100vh;
    background-color: white;
    background-image: url("/../../assets/BG_FGA.png");
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    font-size: calc(10px + 2vmin);
`;

const Base = styled.div`
    background-color: #FFD54F;
    height: 450px;
    width: 800px;
    border-radius: 20px;
    display: flex;
    justify-content: space-around;
`;

const Imagem = styled.div`
    background-color: black;
    height: 450px;
    width: 400px;
    border-top-right-radius: 20px;
    border-bottom-right-radius: 20px;
    background-image: url("../../assets/Logo.jpg");

`;

const Main = styled.div`
    height: 450px;
    width: 400px;
`;

export default function LayoutAutentication({children}){
    return (
        <Background>
            <Base>
                <Main>
                {children}
                </Main>
                <Imagem />
            </Base>
        </Background>
    );
}