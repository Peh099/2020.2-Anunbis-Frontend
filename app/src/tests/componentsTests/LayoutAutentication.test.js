import renderer from 'react-test-renderer';
import LayoutAutentication from '../../components/LayoutAutentication';

describe('Snapshot LayoutAutentication component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<LayoutAutentication />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
