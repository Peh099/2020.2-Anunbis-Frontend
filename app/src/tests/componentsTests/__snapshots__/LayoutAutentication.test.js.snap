// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot LayoutAutentication component matches the snapshot 1`] = `
<div
  className="sc-bdnxRM kFKwhN"
>
  <div
    className="sc-gtsrHT eMPPuQ"
  >
    <div
      className="sc-hKFxyN bIjoSs"
    />
    <img
      alt="Logo"
      className="sc-dlnjwi iYtYcZ"
      src="Logo.jpg"
    />
  </div>
</div>
`;
